### PR TITLE
Add iptable rules to deny VNC access on Compute hosts (bnc #938200)

### PR DESCRIPTION
**Pull request by dguitarbite**
VNC ports are accessible from the management network. This patch
implements iptable rules which will deny access to these ports to all
but the novnc proxy service (nova-controller).

### CODE REVIEW

File: chef/cookbooks/nova/recipes/compute.rb [hunk 1]
```diff
@@ -436,3 +436,12 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 execute "trigger-nova-own-az-config" do
   command "true"
 end
+
+# Set iptables rules for blocking VNC Access for all but the nova-controller node.
+bash 'vnc_access' do
+  code <<-EOH
+    touch /tmp/hello_pranav
+    iptables -I INPUT -p tcp --match multiport --dports 5900:15900 -j DROP
+    iptables -I INPUT -s #{nova_controller_ip} -p tcp --match multiport --dports 5900:15900 -j ACCEPT
```
Comment (mjura) [inline chef/cookbooks/nova/recipes/compute.rb]: This two rules can be merged into one line.

**Comment by mjura** [comment]
As @dirkmueller said, this can be smarter to check on what ports services for `nova` user are listening and in this way configure iptables rules.

**Comment by mjura** [comment]
Why do you want to merge this into `SUSE-Cloud:release/tex/master` proper place should be https://github.com/crowbar/crowbar-openstack and master branch, after changes will be accepted to master we can backport them to tex

**Comment by dguitarbite** [comment]
@mjura The ports are dynamically opened hence, scanning them at run time is not a good idea. If there is a place where this variable is set (mostly hardcoded) under nova, the default values are 5900:15900.

**Comment by dguitarbite** [comment]
@mjura  Updating the location of the PR. Thanks.

**Pull request closed without merging**